IdentityCore/src/util/mac/MSIDKeychainUtil.m

```diff
@@ -63,25 +63,34 @@ - (NSString *)getTeamId
         if (!cfDic)
         {
             MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to retrieve code signing information");
-            CFRelease(selfCode);
-            return nil;
         }
-        
-        NSDictionary* signingDic = CFBridgingRelease(cfDic);
-        keychainTeamId = [signingDic objectForKey:(__bridge NSString*)kSecCodeInfoTeamIdentifier];
-        
-        if (!keychainTeamId)
+        else
         {
-            MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to retrieve team identifier. Using bundle Identifier instead.");
-            NSString *bundleIdentifier = [signingDic objectForKey:(__bridge NSString*)kSecCodeInfoIdentifier];
-            CFRelease(selfCode);
-            return bundleIdentifier;
+            NSDictionary* signingDic = CFBridgingRelease(cfDic);
+            keychainTeamId = [signingDic objectForKey:(__bridge NSString*)kSecCodeInfoTeamIdentifier];
+
+            if (!keychainTeamId)
```

Comment:
just curious, what is the case where kSecCodeInfoTeamIdentifier would be not present but kSecCodeInfoIdentifier is present?

Reply:
not sure, this was the original behavior. Perhaps @rohitnarula7176 can comment?
I'm happy to jump straight to [[NSBundle mainBundle] bundleIdentifier] to retrieve bundle identifier if you think that's more appropriate.

Reply:
I made the change to remove querying bundle ID via kSecCodeInfoIdentifier in the latest change.